frontend/src/components/common/ObjectEventList.tsx

```diff
@@ -13,6 +14,12 @@ export interface ObjectEventListProps {
 export default function ObjectEventList(props: ObjectEventListProps) {
   const [events, setEvents] = useState([]);
 
+  useEffect(() => {
+    if (events) {
+      dispatchHeadlampEvent({ type: HeadlampEventType.OBJECT_EVENT, data: events });
```

Comment:
I'm not sure about having "Headlamp" in the name.

Reply:
The name was included on purpose to not be confused with k8s events 🙂 
Does this make sense?